src/pages/student/main/PasswordReset/index.js

```diff
@@ -4,137 +4,118 @@ import Container from 'react-bootstrap/Container';
+    setSubmitStatus(true);
+    setError('');
+
     try {
       await PasswordResetApi.forgotPassword({ email });
-      showAlertDialog(
-        true,
-        'An email has been sent. Please click the link provided to proceed with the password reset.'
+      setSuccessMessage(
+        'An email has been sent. Please click the link provided in the email sent to proceed with the password reset.'
       );
+      setStatus(true);
     } catch (error) {
-      console.log(error.response);
-      if (error?.response?.data?.errors) {
-        setErrors(error?.response?.data?.errors);
-      } else {
-        showAlertDialog(true, 'An error has occurred.');
-      }
+      setError(error.response.data.error);
+      setSubmitStatus(false);
     }
   };
 
   return (
     <center>
       <Container>
-        <Stack gap={2} className="col-md-5 mx-auto">
+        <Stack gap={2} className='col-md-5 mx-auto'>
           <Form
             onSubmit={handleSubmit(handleOnSubmit)}
             className={style.contentstyle}
           >
-            <div className={style.center} align="start">
-              <center>
-                <Form.Label>
-                  {' '}
-                  <h4> Password Reset </h4>
-                </Form.Label>
-              </center>
-              <Form.Group
-                id={style.Containercentermargin}
-                className="mb - 3"
-                controlId="Email"
-              >
-                <Form.Label>
-                  <h6 style={{ marginBottom: '0px' }}>Email</h6>
-                </Form.Label>
-                <Controller
-                  control={control}
-                  name="email"
-                  defaultValue=""
-                  render={({ field: { onChange, value, ref } }) => (
-                    <Form.Control
-                      onChange={onChange}
-                      value={value}
-                      ref={ref}
-                      className="cntrs"
-                      type="email"
-                      isInvalid={!!errors?.email}
-                      required
-                      maxLength={50}
-                    />
-                  )}
-                />
-                <Form.Control.Feedback type="invalid">
-                  {errors?.email}
-                </Form.Control.Feedback>
-              </Form.Group>
-
-              <center>
-                <Button
-                  type="submit"
-                  onClick={() => {
-                    setShow(false);
-                  }}
-                  id={style.button}
+            {status === false ? (
+              <div className={style.center} align='start'>
+                <center>
+                  <Form.Label>
+                    {' '}
+                    <h4> Password Reset </h4>
+                  </Form.Label>
+                </center>
+                <Form.Group
+                  id={style.Containercentermargin}
+                  className='mb - 3'
```

Comment:
is this not a mistake?
```suggestion
                  className='mb-3'
```